code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -114,15 +110,18 @@
 	fair_market_price = 10
 	payment_department = ACCOUNT_MED
 
+	var/datum/component/gas_connector/connector_component
```

Comment:
I know @LemonInTheDark doesn't like this method of using components, as they're generally meant to be isolated (and also it hangs a reference) 

That being said I don't know what's the best way to avoid using this in this context, so I'll leave it to them to see if they have any suggestions. (Maybe just grab a weakref to the internal connector in Initialize?)

Reply:
if you want a direct reference to something just make it a dang datum. This has no reason to be a component, it doesn't WANT to be a component any usage would need a direct ref and to directly read vars off it

Reply:
So I'll just transform it to /datum/gas_connector ?